pkg/logging/logkey/constants.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logkey
```

Comment:
What about putting just this and the yaml in `pkg/logconfig` (and then callers would import `knative/pkg/logging` and `knative/eventing/pkg/logconfig`)?

Reply:
you may hate the new change more... Lets see. I moved the relevant keys to apis.